### PR TITLE
Enable product tests for Cassandra

### PR DESCRIPTION
**Pull request by maciejgrzybek**
Only Java-base tests are enabled (no convention-base tests are supported) for now.

---

Internal TD review: https://github.com/Teradata/presto/pull/409

### CODE REVIEW

**Comment by electrum** [comment]
I'm concerned this will increase product test runtime, which is already the longest running Travis job. How can we improve this?

**Comment by maciejgrzybek** [comment]
We can create a separate job for Cassandra, as we did for Accumulo.

But for now it should not take much longer given startup time for Cassandra is small. We also don't have many (just one) product test for Cassandra now. Rest of the resources and infra for product tests in Cassandra (e.g. Hive etc.) is reused.

Maybe we should have separate jobs per group of tests? At least for bigger groups like Hive, JDBC/MySql/Postgres? Eventually we would have separate task per module/connector and maybe some complex tests, going beyond boundaries of single connector (e.g. query federation).

Some raw numbers (unfortunately running time for product tests is not super stable, so cannot compare times very legitimately, because of big variance):
last build from master took 31min 2s, here, with Cassandra 32min 30s.

To sum up, I'd keep adding product tests here, as long as we reach significant test suite. Then it's worth extracting.
Benefit of having them in one job is that spinning the infrastructure is happening only once for all product tests.
So, savings on executing product tests independently should overcome overhead on spinning infra more than once.

**Comment by electrum** [comment]
I agree, it's not a problem now, since there's only one test and the setup takes may be a minute, but I want to keep this on the radar as something to improve later.

**Comment by electrum** [comment]
Merged, thanks!
